Makefile

```
@@ -70,7 +70,12 @@ vmlinux: linux.config
 .PHONY: run
 run: initrd bzImage
 	$(MAKE) generate
-	go run . boot -c 4
+	go run . boot -c 4 -i "../initrd"
```

I think the initrd is created in the root of the gokvm repository.
```suggestion
	go run . boot -c 4 -i "./initrd"
```

---

done